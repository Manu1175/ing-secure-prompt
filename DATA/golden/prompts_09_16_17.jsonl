{"id": "prompts_09_16_17:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is the Loan Approval Policy still active?", "expected_scrub": "Is the <POLICY_NAME> still active?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the expiry date and CCV for Jane Clark?", "expected_scrub": "Can you confirm the <EXPIRY_DATE> and <CCV> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Jane Clark and her Current Account?", "expected_scrub": "What is the <RELATIONSHIP_TYPE> for <CUSTOMER_NAME> and their <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the password for Emily Clark?", "expected_scrub": "What is the <PASSWORD> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "When was the Privacy Protection Policy last reviewed?", "expected_scrub": "When was the <POLICY_NAME> last reviewed?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is Mobile Banking for Tom Smith currently active?", "expected_scrub": "Is <PRODUCT_NAME> for <CUSTOMER_NAME> currently <STATUS>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What category does the KYC Policy fall under?", "expected_scrub": "What category does the <POLICY_NAME> fall under?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the PIN for Michael Garcia?", "expected_scrub": "What is the <PIN> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Who approves the Customer Service Policy?", "expected_scrub": "Who approves the <POLICY_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "When was the last interaction for Chris Taylor’s Current Account?", "expected_scrub": "When was the last interaction for <CUSTOMER_NAME>’s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is the password for Emily Clark strong?", "expected_scrub": "Is the <PASSWORD> for <CUSTOMER_NAME> strong?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "How often is the IT Governance Policy reviewed?", "expected_scrub": "How often is the <POLICY_NAME> reviewed?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Who holds the Savings Account for Laura Walker?", "expected_scrub": "Who holds the <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the CCV for Tom Davis’s credit card?", "expected_scrub": "What’s the <CCV> for <CUSTOMER_NAME>’s credit card?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is the Fraud Detection Policy still in effect?", "expected_scrub": "Is the <POLICY_NAME> still in effect?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is Emily Jones a subscriber to Travel Insurance?", "expected_scrub": "Is <CUSTOMER_NAME> a subscriber to <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Verify the credit card number for Jane Clark.", "expected_scrub": "Verify the <CREDIT_CARD_NUMBER> for <CUSTOMER_NAME>.", "entities": null, "notes": null}
{"id": "prompts_09_16_17:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Who is responsible for the Operational Risk Policy?", "expected_scrub": "Who is responsible for the <POLICY_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "When did Anna Walker start her Current Account?", "expected_scrub": "When did <CUSTOMER_NAME> start their <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and email for Emily Taylor?", "expected_scrub": "What is the <MASKED_PIN> and <EMAIL> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is the Loan Approval Policy still active?", "expected_scrub": "Is the <POLICY_NAME> still active?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Can you confirm the expiry date and CCV for Jane Clark?", "expected_scrub": "Can you confirm the <EXPIRY_DATE> and <CCV> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the relationship type for Jane Clark and her Current Account?", "expected_scrub": "What is the <RELATIONSHIP_TYPE> for <CUSTOMER_NAME> and their <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the password for Emily Clark?", "expected_scrub": "What is the <PASSWORD> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "When was the Privacy Protection Policy last reviewed?", "expected_scrub": "When was the <POLICY_NAME> last reviewed?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is Mobile Banking for Tom Smith currently active?", "expected_scrub": "Is <PRODUCT_NAME> for <CUSTOMER_NAME> currently <STATUS>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What category does the KYC Policy fall under?", "expected_scrub": "What category does the <POLICY_NAME> fall under?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the PIN for Michael Garcia?", "expected_scrub": "What is the <PIN> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Who approves the Customer Service Policy?", "expected_scrub": "Who approves the <POLICY_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "When was the last interaction for Chris Taylor’s Current Account?", "expected_scrub": "When was the last interaction for <CUSTOMER_NAME>’s <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is the password for Emily Clark strong?", "expected_scrub": "Is the <PASSWORD> for <CUSTOMER_NAME> strong?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "How often is the IT Governance Policy reviewed?", "expected_scrub": "How often is the <POLICY_NAME> reviewed?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Who holds the Savings Account for Laura Walker?", "expected_scrub": "Who holds the <PRODUCT_NAME> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What’s the CCV for Tom Davis’s credit card?", "expected_scrub": "What’s the <CCV> for <CUSTOMER_NAME>’s credit card?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is the Fraud Detection Policy still in effect?", "expected_scrub": "Is the <POLICY_NAME> still in effect?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Is Emily Jones a subscriber to Travel Insurance?", "expected_scrub": "Is <CUSTOMER_NAME> a subscriber to <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Verify the credit card number for Jane Clark.", "expected_scrub": "Verify the <CREDIT_CARD_NUMBER> for <CUSTOMER_NAME>.", "entities": null, "notes": null}
{"id": "prompts_09_16_17:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "Who is responsible for the Operational Risk Policy?", "expected_scrub": "Who is responsible for the <POLICY_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "When did Anna Walker start her Current Account?", "expected_scrub": "When did <CUSTOMER_NAME> start their <PRODUCT_NAME>?", "entities": null, "notes": null}
{"id": "prompts_09_16_17:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_09_16_17.xlsx", "modality": "text", "c_level": null, "raw_text": "What is the masked PIN and email for Emily Taylor?", "expected_scrub": "What is the <MASKED_PIN> and <EMAIL> for <CUSTOMER_NAME>?", "entities": null, "notes": null}
