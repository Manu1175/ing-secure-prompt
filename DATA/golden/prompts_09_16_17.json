[
  {
    "text": "Is the Loan Approval Policy still active?",
    "labels": []
  },
  {
    "text": "Can you confirm the expiry date and CCV for Jane Clark?",
    "labels": []
  },
  {
    "text": "What is the relationship type for Jane Clark and her Current Account?",
    "labels": []
  },
  {
    "text": "What is the password for Emily Clark?",
    "labels": []
  },
  {
    "text": "When was the Privacy Protection Policy last reviewed?",
    "labels": []
  },
  {
    "text": "Is Mobile Banking for Tom Smith currently active?",
    "labels": []
  },
  {
    "text": "What category does the KYC Policy fall under?",
    "labels": []
  },
  {
    "text": "What is the PIN for Michael Garcia?",
    "labels": []
  },
  {
    "text": "Who approves the Customer Service Policy?",
    "labels": []
  },
  {
    "text": "When was the last interaction for Chris Taylor’s Current Account?",
    "labels": []
  },
  {
    "text": "Is the password for Emily Clark strong?",
    "labels": []
  },
  {
    "text": "How often is the IT Governance Policy reviewed?",
    "labels": []
  },
  {
    "text": "Who holds the Savings Account for Laura Walker?",
    "labels": []
  },
  {
    "text": "What’s the CCV for Tom Davis’s credit card?",
    "labels": []
  },
  {
    "text": "Is the Fraud Detection Policy still in effect?",
    "labels": []
  },
  {
    "text": "Is Emily Jones a subscriber to Travel Insurance?",
    "labels": []
  },
  {
    "text": "Verify the credit card number for Jane Clark.",
    "labels": []
  },
  {
    "text": "Who is responsible for the Operational Risk Policy?",
    "labels": []
  },
  {
    "text": "When did Anna Walker start her Current Account?",
    "labels": []
  },
  {
    "text": "What is the masked PIN and email for Emily Taylor?",
    "labels": []
  }
]